.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
+      - name: Get tag
+        id: tag
+        uses: dawidd6/action-get-tag@v1
+      - uses: actions/checkout@v2
+      - uses: ncipollo/release-action@v1
+        with:
+          github_token: ${{ steps.github_app_token.outputs.token }}
+          body: "Functional tests for OpenSearch Dashboards and OpenSearch Dashboards Plugins ${GITHUB_REF/refs\/tags\//}"
```

Comment:
Q: is it supposed to read the tag of `HEAD` by `${GITHUB_REF/refs\/tags\//}`?

Reply:
should it be `steps.tag.outputs.tag`? coming from example https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/auto-release.yml#L29

Reply:
Yes `${GITHUB_REF/refs\/tags\//}` this would get the tag name, please check from hat I found
https://github.com/orgs/community/discussions/26686#discussioncomment-3252853, if we decide to remove this `dawidd6/action-get-tag` then we just have `${GITHUB_REF/refs\/tags\//}`.